DigitalLearningSolutions.Web/Controllers/TrackingSystem/Centre/Administrator/AdministratorController.cs

```diff
@@ -156,12 +163,19 @@ public IActionResult DeactivateAdmin(int adminId, int? returnPage)
         [ServiceFilter(typeof(VerifyAdminUserCanAccessAdminUser))]
         public IActionResult DeactivateAdmin(int adminId, DeactivateAdminViewModel model)

```

Comment:
As deletion can be a potentially serious thing, would it be worth renaming this method (and the get) to DeactivateOrDeleteAdmin just for developer's benefit/safety?